src/AppContainer.dev.js

```diff
@@ -69,7 +69,7 @@ class AppContainer extends Component {
     if (error) {
       return <this.props.errorReporter error={error} />;
     } else {
-      return <this.props.component {...this.props.props} />;
+      return React.cloneElement(this.props.children);
```

Comment:
Let’s return `React.Children.only(this.props.children)` instead?
This way we enforce that there’s a single child.
`React.cloneElement()` isn’t necessary here.